### PR TITLE
feat(Packages): import orbit-design-tokens

### PR DESCRIPTION
**Pull request by vepor**
Didn’t change any implementation, except the fix for TS definitions. Other than that, few notes: 

- I needed to temporarily suppress some ESLint warnings/error, because it doesn’t make sense to refactor it right now – will be deleted or refactored anyway.
- Also I’ve created separate `build` script, so it can be used locally and also on the CI, via `build-ci`  and `test-ci`. It was also necessary to build the tokens first.

Also, please just merge and don't squash.<br/><br/><br/><url>LiveURL: https://orbit-feat-import-tokens.surge.sh</url>

### CODE REVIEW

**Comment by silvenon** [comment]
You need to add `orbit-design-tokens`' `build` script in `build-ci`.

File: packages/orbit-design-tokens/src/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,623 @@
...
+type CustomBase = Partial<Base>;
+
+type CustomFoundation = Partial<{
+  palette: CustomPalette;
+  base: CustomBase;
+}>;
+
+export declare const ConvertHexToRgba: (color: string, opacity: number) => string;
+export declare const FromPlainObject: (themePaletteColors: ThemePaletteColors) => Tokens;
+export declare const GetTokens: (customFoundation?: CustomFoundation) => Tokens;
```
Comment (filipdanisko) [inline packages/orbit-design-tokens/src/index.d.ts:622]: Just a quick fix of exported types, did not wanted to do that in the orbit-design-tokens repository
```suggestion
export declare const fromPlainObject: (themePaletteColors: ThemePaletteColors) => Tokens;
export declare const getTokens: (customFoundation?: CustomFoundation) => Tokens;
```

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -41,7 +41,7 @@ jobs:
           key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock', '.yarnclean') }}
 
       - name: Install dependencies
-        run: yarn install && yarn bootstrap
+        run: yarn install && yarn bootstrap && yarn orbit-design-tokens build
```
Comment (silvenon) [inline .github/workflows/test.yml:44]: Better add another step below named `Build design tokens` for now, just to be explicit about it, later we will work on adding more sense to our scripts.

File: packages/orbit-design-tokens/.gitignore [hunk 3]
```diff
@@ -0,0 +1,5 @@
+/node_modules
+.idea
+es
+lib
+.vscode
```
Comment (silvenon) [inline packages/orbit-design-tokens/.gitignore:5]: You can remove `.node_modules`, `.vscode` and `.idea`, they are already covered by our root `.gitignore`.

File: packages/orbit-design-tokens/.npmrc [hunk 4]
```diff
@@ -0,0 +1,2 @@
+scope=@kiwicom
+access=public
```
Comment (silvenon) [inline packages/orbit-design-tokens/.npmrc:2]: Deleting this file should be fine, but just in case add `"private": false` to `package.json`.

**Review by silvenon — CHANGES_REQUESTED** [review]

**Review by silvenon — APPROVED** [review]

**Pull request merged**